aiida/cmdline/params/options/__init__.py

```diff
@@ -151,7 +151,7 @@ def active_process_states():
 RAW = OverridableOption('-r', '--raw', 'raw', is_flag=True, default=False,
                         help='display only raw query results, without any headers or footers')
 
-HOSTNAME = OverridableOption('-H', '--hostname', help='hostname')
+HOSTNAME = OverridableOption('-H', '--hostname', help='the computer hostname')
```

Comment:
Keep in mind this is also used for the postgres host name

Reply:
ok, I revert this back, anyway I put a longer one for verdi computer setup